src/app/pages/office/cat-adi-part2/components/eco-related-fault/eco-related-fault.html

```diff
@@ -0,0 +1,31 @@
+<ion-row [formGroup]="formGroup" class="mes-validated-row">
+    <div class="validation-bar" [class.ng-invalid]="invalid"></div>
+    <ion-col class="component-label ion-align-self-start" size="32">
+        <ion-row class="spacing-row"></ion-row>
+        <label>Fault related to ECO capture</label>
+    </ion-col>
+    <ion-col size="63" class="ion-align-self-center ion-padding-start">
+        <ion-row class="spacing-row"></ion-row>
+        <ion-row>
+            <ion-select id="eco-fault-selector"
+                        okText="Submit"
+                        cancelText="Cancel"
+                        placeholder="Select fault"
+                        class="black-text"
+                        formControlName="ecoRelatedFault"
+                        [class.ng-invalid]="invalid"
+                        (ionChange)="ecoFaultChanged($event.target.value)"
+                        [interfaceOptions]="{cssClass:'mes-select', enableBackdropDismiss:false}">
+                <ion-select-option *ngFor="let fault of drivingFaults"
```

Comment:
We could use a trackByIndex which is a recommended on ngFor loops for performance purposes